git-stash.txt

```diff
@@ -0,0 +1 @@
+
```

Comment:
See if you can create the file with content directly from the command line. The exercises suggest that you run `git help git-stash >> git-stash.txt`. That should give you some text that you can add to Git and commit.